### ISSUE TITLE
Content Explorer popup menu appears in wrong location

### ISSUE THREAD

**Issue by scherroman** [box/box-ui-elements#183]
### Steps
I am using the [Box Content Explorer](https://developer.box.com/docs/box-content-explorer) React element in a [Semantic UI React Modal](https://react.semantic-ui.com/modules/modal) element.

Using [box-ui-elements 3.6.1](https://github.com/box/box-ui-elements) and [semantic-ui-react 0.78.2](https://github.com/Semantic-Org/Semantic-UI-React).

### Expected Behavior
The triple dot popup menu appears directly below the cursor after clicking the triple dot for a file.

### Actual Behavior
The triple dot popup menu appears far below and off to the side of the cursor.

![print marker file explorer random menu location](https://user-images.githubusercontent.com/7923938/36226936-58d3f6d0-119d-11e8-9fb0-38119d3b3be2.JPG)

The larger the screen size, the further below and off to the side the popup appears.

### Test Case

[Here's a Codepen showing this](https://codepen.io/scherroman/pen/YeeyYL), using Box UI Elements 3.4.1 explorer.js  and Semantic UI 2.2.14 .js

**Comment by tonyjin** [comment]
Thanks for the report, we'll take a look.

**Comment by priyajeet** [comment]
This will be a tricky one to fix as one solution may break another use case. Unfortunately don't really have a proper solution for this at the moment. We use Tether.js/react-tether which is sensitive to the stacking context and is not perfect. Fixed/absolute positioning by other 3rd party modals may interfere with it. We have experimented with popper.js as a replacement, which has its own problems. Will need some time to investigate.
